server/config/validation/add-panel-members.spec.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable */
+(function() {
+    'use strict';
+  
+    var validate = require('validate.js')
+      , validator = require('./generate-panel')
+      , validatorResult = null;
```

Comment:
could we get this using const and let instead of var?

Reply:
I have changed this now